modules/serverless-creating-broker-annotation.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-creating-broker-annotation_{context}"]
+= Creating a broker by annotating a trigger
+
+[IMPORTANT]
+====
+If you create a broker by using the `eventing.knative.dev/injection=enabled` annotation, you cannot delete this broker without cluster administrator permissions.
+If you delete the broker without having a cluster administrator remove this annotation first, the broker will be created again after deletion.
+====
+
+.Prerequisites
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+
+.Procedure
+* Create a `Trigger` YAML object that has the `eventing.knative.dev/injection=enabled` annotation:
++
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1beta1
+kind: Trigger
+metadata:
+  annotations:
+    injection: enabled
```

Comment:
The annotation name here needs to be `eventing.knative.dev/injection` as the rest of examples through `oc label` etc.
```suggestion
    eventing.knative.dev/injection: enabled
```

Reply:
Hi @abrennan89, this suggestion has not been incorporated, is this intentional?

Reply:
I thought I accepted the suggestion but it didn't seem to work, updated now in the next commit, thanks